pylintrc

```diff
@@ -209,7 +209,7 @@ max-attributes=7
 min-public-methods=2
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=20
+max-public-methods=22
```

Comment:
Ideally I'd set the max public methods for that one class to 21 temporarily. Reconsidering, maybe a global change is a bad idea. Opting instead for a local one.